packages/kbn-config-schema/README.md

```diff
@@ -243,7 +243,7 @@ __Output type:__ `{ [K in keyof TProps]: TypeOf<TProps[K]> } as TObject`
 __Options:__
   * `defaultValue: TObject | Reference<TObject> | (() => TObject)` - defines a default value, see [Default values](#default-values) section for more details.
   * `validate: (value: TObject) => string | void` - defines a custom validator function, see [Custom validation](#custom-validation) section for more details.
-  * `unknowns: 'allow' | 'ignore' | 'forbid'` - indicates whether unknown object properties should be allowed, ignored, or forbidden. It's `forbid` by default.
+  * `unknowns: 'allow' | 'ignore' | 'forbid'` - indicates whether unknown object properties and sub-properties should be allowed, ignored, or forbidden. It's `forbid` by default. 
```

Comment:
I was thinking, for completeness, we can also mention the `skipUnknownKeys` you added [here](https://github.com/elastic/kibana/pull/187006/files#diff-5d73aa751d588d3af829b9c4ae821d3bddc0b62b813ef4144b3927098960a2c7R43-R48) on line 67 of the README where we talk about the global `validate` method and it's options.

Reply:
Great point! I'll add that bit!

Reply:
Done in [`08a0ace` (#187006)](https://github.com/elastic/kibana/pull/187006/commits/08a0ace0810c7cec77083f2efdeeef14610532eb)